apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -182,6 +211,22 @@ var TopInstructions = React.createClass({
     win.focus();
   },
 
+  handleHelpTabClick(){
+    this.setState({helpTabVisible: true});
+    $(".helpTab").css({color: "#333"});
+    $(".helpTab").css({borderBottom: "2px solid #333"});
+    $(".instructionsTab").css({color: color.charcoal});
+    $(".instructionsTab").css({borderBottom: ""});
+  },
+
+  handleInstructionTabClick(){
+    this.setState({helpTabVisible: false});
+    $(".helpTab").css({color: color.charcoal});
+    $(".helpTab").css({borderBottom: ""});
+    $(".instructionsTab").css({color: "#333"});
+    $(".instructionsTab").css({borderBottom: "2px solid #333"});
+  },
+
```

Comment:
Modifying styles dynamically like this should be avoided whenever possible in React components. Instead, in your render method you should set the style based on the state.

For example, I would expect you to perhaps have a style that is something like `styles.selectedTab = { color: '#333', borderBottom: '2px solid #333'}`

Then in your JSX you can have:
```jsx
<a
  style={{
    ...this.styles.helpTab,
    ...(this.state.helpTabSelected && styles.selectedTab
  }}
/>
```

One of the goals of React is to have everything be declarative as much as possible. This means you should be able to render your component entirely based off of state/props. When you start dynamically modifying elements such as this, it now also depends on what sorts of dynamic actions you've taken, making things more difficult to reason about.